TaskfMRIAnalysis/TaskfMRIAnalysis.sh

```diff
@@ -39,56 +39,92 @@
 #
 #~ND~END~
 
+set -eu
```

Comment:
Is this just for temporary debugging purposes?  At some point, we hope that the `debug.shlib` library will handle error trapping.  At a minimum, would be good to put this line immediately in front of the sourcing of `debug.shlib`.  See `DeDriftAndResample` for a recent example where we had to do this.

Reply:
`set -eu` is intentionally part of the newopts template, it catches errors from typoing a variable name or using a variable before it is set, which the debug trap can't catch.  It is at the top of the file, and not too many lines from the `source` lines, unlike the situation in DeDrift where the `source` lines are at the bottom of the file, so I don't expect as much potential here for copy paste trouble.